server/src/test/java/com/objectcomputing/checkins/endtoend/pages/HomePage.java

```diff
@@ -1,72 +1,79 @@
 /**
- * 
+ *
  */
 package com.objectcomputing.checkins.endtoend.pages;
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import java.io.File;
+import java.net.URL;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 /**
  * @author tajr
- * 
+ *
  * This class will store all the locators and methods of home page.
  *
  */
 public class HomePage {
-	
-	WebDriver driver;
-	
-	public HomePage(WebDriver ldriver)
-	{
-		this.driver = ldriver;
-	}
-	
-	@FindBy(xpath="/html[1]/body[1]/div[1]/form[1]/fieldset[1]/label[1]") 
-	WebElement chooseafile_btn;
-	
-	@FindBy(id="file")
-	WebElement filepath;
-	
-	@FindBy(name="submit")
-	WebElement upload_btn;
 
-	@FindBy(className="success")
-	WebElement success_msg;
-	
-	@FindBy(className="error")
-	WebElement error_msg;
-	
-	
+    WebDriver driver;
+    @FindBy(xpath = "/html[1]/body[1]/div[1]/form[1]/fieldset[1]/label[1]")
```

Comment:
Why do all of these have `[1]`? instead of being `[0]`?

Reply:
xpath indexes aren't 0 based...they start with 1 🤷‍♂️